.gitignore

```diff
@@ -11,3 +11,4 @@ pull_dev_database.sh
 
 snapshots
 .env*
+!.env.*.example
```

Comment:
i think a better pattern would be commiting a `.env.example` file in version control, in the same way we do for example with the docs api: https://github.com/WPMedia/newsroom-documents-api/blob/develop/.env.example

with that in place, rather than copying all the necessary env vars into the README steps, we can just direct people to `cp .env.example .env.<local/dev/prod>` and fill in the values via secrets manager and/or keybasing a teammate